playerctrl/handlers.go

```diff
@@ -161,43 +161,55 @@ func (ctrl *Controller) handleSeek(ev events.Event) {
 	log.Infof("%v: type={%v} id={%v}", msg, ev.Type, ev.GroupID)
 }
 
-func (ctrl *Controller) handleSynchronize(ev events.Event) {
+func (ctrl *Controller) handleSetSynchronized(ev events.Event) {
 	msg := "[playerctrl] handle set synchronized"
 	ctx := context.Background()
 	sessionID := string(ev.GroupID)
 
-	payload, ok := ev.Data.(SynchronizePayload)
+	payload, ok := ev.Data.(SetSynchronizedPayload)
 	if !ok {
 		log.Errorf("%v: %v", msg, ErrEventPayloadMalformed)
 		return
 	}
 	userID := payload.UserID
+	synchronized := payload.Synchronized
+	// notify sse that user change sync status
+	defer ctrl.eventBus.Publish(
+		sse.UserSynchronizedChange,
+		events.GroupID(sessionID),
+		sse.UserSynchronizedChangePayload{Synchronized: synchronized},
```

Comment:
maybe it makes sense to move the db query in `handleSentSynchronized`

Reply:
That's the way it was initially in handleSynchronized, but it was changed because the events are asynchronous and the http handlers might need to return something after the db query is executed and can't wait on the event handlers to finish.